lang/en/website-notice.yaml

```diff
@@ -0,0 +1,12 @@
+banner:
+  title: Bitcoincash.org and its owners no longer accurately represent Bitcoin Cash!
+  sub: Click to find out more
+
+title: Notice to services that use Bitcoin Cash (BCH)
+h1: <span class="text-muted">Bitcoincash.org</span> is unreliable for information regarding the BCH network
+h2: Recommended actions
+p1: Due to the decentralized nature of the BCH network, there is no official reference website for general Bitcoin Cash information. Unfortunately, the most popular of those informational websites, bitcoincash.org, has been publishing information that does not reflect the current reality of the BCH network and the November 15th 2020 scheduled upgrade.
+p2: Bitcoin ABC, which owns bitcoincash.org, wants to implement an 8% miner tax. This proposal has been rejected by the vast majority of the Bitcoin Cash ecosystem, and Bitcoin ABC will likely fork to a minority chain as a new coin. This reality has already been recognized in statements from Bitgo, Coinbase, Kraken, and others. Unfortunately, Bitcoin ABC is still presenting its software as the only compatible option for the November 15th 2020 upgrade.
+p3: Please replace any references to bitcoincash.org with an alternative, such as <a href="/$( _lang_ )$">bch.info</a>.
+p4: 'In order to stay on the Bitcoin Cash network, BCH services need to replace any Bitcoin ABC nodes with an alternative. Bitcoin Cash Node (BCHN) is a drop-in replacement for Bitcoin ABC and is strongly recommended for miners and pools. There are five other BCH nodes available at this time that have various benefits: BCHD, Bitcoin Unlimited, Bitcoin Verde, Flowee and Knuth.'
+p5: Also, make sure your node is up to date so that it’s compatible with the November 15th 2020 upgrade. This upgrade will make Bitcoin Cash block times more reliable.
```

Comment:
> make sure your node is up to date

This is the only text to use "you" directly so it sticks out a little. There is also a contraction "it's" which is informal compared to the rest of the text. I recommend:

> please make sure nodes are up to date so that they will be compatible with ...